Transformations/Models.MetaTransformation/Meta/Class2Type.cs

```diff
@@ -889,11 +886,11 @@ protected virtual CodeMemberMethod CreateGetCollectionForFeature(IClass input, C
                 return getCollectionForFeature;
             }
 
-            private CodeMemberMethod AddToCollectionsForFeature(CodeMemberMethod method, ITypedElement feature, ITransformationContext context)
+            private CodeMemberMethod AddToCollectionsForFeature(CodeMemberMethod method, ITypedElement feature, CodeMemberProperty property, ITransformationContext context)
             {
                 if (feature.UpperBound != 1)
                 {
-                    var propRef = new CodePropertyReferenceExpression(new CodeThisReferenceExpression(), "_" + feature.Name.ToCamelCase());
```

Comment:
This really is a place that is flawed, because the assumption that the property name is the feature name can be wrong, so here, the extra argument really is necessary, yes.